core/src/main/java/org/stellar/anchor/util/MemoHelper.java

```diff
@@ -48,14 +48,14 @@ public static MemoType memoType(Memo memo) {
   }
 
   public static Memo makeMemo(String memo, String memoType) throws SepException {
-    if (isEmpty(memoType)) {
-      memoType = "none";
-    }
-
     if (isEmpty(memo)) {
       return null;
     }
 
+    if (isEmpty(memoType)) {
+      throw new SepValidationException("memo_type is required if memo is specified");
+    }
```

Comment:
This is a unrelated change, @lijamie98 let me know if you think I should revert. We should probably require memo type to be specified instead of assuming the type is none, because the error message returned will assume memo type was specified.

Reply:
I think we can keep your change that throws exception unless there are cases that we must return `none`.